pkg/validate/serialize.go

```diff
@@ -17,12 +21,35 @@ func Verify(yamlFileName string) error {
 	}
 
 	// Value returned is a marshaled go type (CSV Struct).
-	_, err = unmarshal(rawYaml)
+	csv, err := unmarshal(rawYaml)
 	if err != nil {
 		return fmt.Errorf("Error unmarshalling YAML to OLM's csv type for %s file:  #%s ", yamlFileName, err)
 	}
 
-	fmt.Printf("%s is verified", yamlFileName)
+	// Flag for checking if there's any mandatory field missing.
+	isMandatoryFieldMissing := false
+
+	// Contains error logs for all missing optional and mandatory fields.
+	log := csvInspect(csv)
+
+	sort.SliceStable(log, func(i, j int) bool {
+		return log[i] > log[j]
+	})
+
+	for i := range log {
+		if strings.Contains(strings.ToLower(log[i]), "error") {
+			isMandatoryFieldMissing = true
+		}
+		fmt.Println(log[i])
+	}
+
+	if isMandatoryFieldMissing {
+		fmt.Printf("Populate all the mandatory fields missing from %s file.", yamlFileName)
+		os.Exit(1)
```

Comment:
if you are going to exit the function without returning an `error`, might as well remove that from the function definition and have it not return anything.

Reply:
Not making this change in the CLI in this PR.